docs/en/05_Releases_and_changelogs/cwp_2.6.0.md

```diff
@@ -0,0 +1,49 @@
+# 2.6.0
+
+## Overview
+
+This release includes CMS and Framework version 4.6.0
+
+- [CMS Recipe 4.6.0](https://docs.silverstripe.org/en/4/changelogs/4.6.0/)
+
+Upgrading to Recipe 2.6.0 is recommended for all CWP sites. This upgrade can be carried out by any development team familiar with the Silverstripe CMS. However, if you would like Silverstripe and the CWP team's assistance, you can request support via the [Service Desk](https://www.cwp.govt.nz/service-desk/new-request/).
+
+## New features
+
+The [release announcement](https://www.cwp.govt.nz/updates/news/cwp-2-6-has-arrived) includes the note worthy features, but be sure to review the change log for full detail of all new features.
+
+## Solr no longer indexes draft content
+
+This CWP release includes an update to the [fulltextsearch module](https://github.com/silverstripe/silverstripe-fulltextsearch) which now has more secure defaults.  Most notably, draft content is now no longer added to the solr index by default.  After upgrading your website, ensure that you run Solr_Reindex on production to remove previously indexed content that should no longer be there.
+
```

Comment:
I believe it's worth mentioning the new `canView() check against an anonymous user` in the changelog as well

Reply:
done